build/package/docker/atlascli-centos7-rpm.Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM centos:7
+
+ARG revision
+ARG created_at
+ARG version
+
+ENV URL=https://mongodb-mongocli-build.s3.amazonaws.com/mongocli-master/dist/${revision}_${created_at}/mongodb-atlas-cli_${version}-next_linux_x86_64.rpm
```

Comment:
[q] do we need these Dockerfiles? Can't we make `URL` a `ARG` and pass it before scheduling the docker image run?

Reply:
or just a arg which would decide if Dockerfile would run in `atlascli` or `mongocli` mode?

Reply:
updated